file/buf/buf.go

```diff
@@ -38,6 +38,14 @@ type Options struct {
 	// Writes will be compressed but reads will read raw
 	// compressed bytes.
 	Compress bool
+
+	// CompressType type of compression used on the file
+	// best speed, best compression, default
+	CompressType int
```

Comment:
CompressionLevel instead? 

Could you comment what the values are and what they mean?